src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineParser.java

```diff
@@ -71,6 +73,14 @@
     public static final String COMMENT = "#";
     public static final String POSITIONAL_ARGUMENTS_NAME = "Positional Argument";
 
+    // Map from (full class) name of each GATKCommandLinePluginDescriptor requested and
+    // found to the actual descriptor instance
+    Map<String, GATKCommandLinePluginDescriptor<?>> pluginDescriptors = new HashMap<>();
```

Comment:
Can this be private rather than package-protected (since it has a public accessor)?